alica_engine/include/engine/RunnableObject.h

```diff
@@ -37,11 +39,13 @@ class RunnableObject
     const std::string& getName() { return _name; };
     AlicaTime getInterval() const { return _msInterval; };
     void setInterval(int32_t msInterval) { _msInterval = AlicaTime::milliseconds(msInterval); };
-    bool getRequiresParameters() const { return _requiresParameters; }
-    void setRequiresParameters(bool requiresParameters) { _requiresParameters = requiresParameters; }
-    void stop();
+    bool getInheritBlackboard() const { return _blackboardBlueprint == nullptr; };
```

Comment:
What about for master plan (doesn't inherit, but no blackboard blueprint)

Reply:
For a master plan this could return wrong results. Where the blackboard is used (RunnableObject::start, RunnableObject::stop) this is combined with a check if there is a parent.